rentsafe/rent_views/clients.py

```diff
@@ -48,7 +49,7 @@ def clients_credit_dashboard(id):
     taken_credits_color_totals = {}
 
     rate_ = 1
-    if rate_obj := LeaseCurrencyRate.objects.filter(company_id= client_id).first():
+    if rate_obj := CurrencyRate.objects.filter(user = client_id).first():
```

Comment:
Alter this to  compare company ids not user
```suggestion
    if rate_obj := CurrencyRate.objects.filter(user__company = client_id).first():
```